pkg/trace/agent/agent.go

```diff
@@ -12,6 +12,7 @@ import (
 	"time"
 
 	"github.com/DataDog/datadog-agent/pkg/obfuscate"
+	"github.com/DataDog/datadog-agent/pkg/serverless/trace/inferredspan"
```

Comment:
Up to now, we've managed to never import serverless package in non-serverless package.
I think you can hook to `modifySpan` as I think the only input you need is the span?